doc/quickstart.md

```diff
@@ -14,23 +14,27 @@ See [Setup](./setup.md) for other ways and more details on setting up Lean.
     del elan-init.ps1
     ```
     See the [elan repo](https://github.com/leanprover/elan) for other installation options and details.
```

Comment:
Should we change this to installing elan from the VS Code extension?

Reply:
Done.